content/docs/getting_started/upgrade.md

```diff
@@ -103,6 +103,44 @@ Omit the `--values` flag if you prefer to use the default settings.
 
 Run `helm upgrade --help` for more options.
 
+## Upgrading Third Party Dependencies
+
+### Envoy
+
+The envoy version can be updated by changing the value of the `envoyImage` variable in the osm-mesh-config. For instance, to update the envoy-alpine image to v1.19.1, the following command should be run: 
+
+```bash
+kubectl patch meshconfig osm-mesh-config -n osm-system -p '{"spec":{"sidecar":{"envoyImage":"envoyproxy/envoy-alpine:v1.19.1"}}}' --type=merge
+```
```

Comment:
Could you add a note to request users to specify an image digest so they are not vulnerable to supply chain attacks. E.g. `envoyImage: envoyproxy/envoy-alpine@sha256:6502a637c6c5fba4d03d0672d878d12da4bcc7a0d0fb3f1d506982dde0039abd`

Reply:
@shashankram same thing for Prometheus, Grafana, and Jaeger?